src/runtime/runtime.js

```diff
@@ -888,6 +903,7 @@ function createPublicRuntime(runtime) {
     attachSmartButton: runtime.attachSmartButton.bind(runtime),
     getPropensityModule: runtime
         .getPropensityModule.bind(runtime),
+    eventManager: runtime.eventManager.bind(runtime),
```

Comment:
based on comments in previous review, this should not be here.

Reply:
Ok